go.mod

```diff
@@ -431,4 +431,5 @@ replace (
 	github.com/redis/go-redis/v9 => github.com/gravitational/redis/v9 v9.0.2-teleport.2
 	github.com/sijms/go-ora/v2 => github.com/gravitational/go-ora/v2 v2.0.0-20230821114616-e2a9f1131a46
 	github.com/vulcand/predicate => github.com/gravitational/predicate v1.3.1
+	github.com/gravitational/license => github.com/Envek/license v0.0.0-20231209003859-ae0c37e57823
```

Comment:
TODO: get rid of this replace after https://github.com/gravitational/license/pull/10 is merged